src/Web/FlashAlerts.hs

```diff
@@ -15,14 +13,14 @@ getFlashes = do
   let info    = maybe "" infoTemplate maybeInfo
   pure $ err <> info
 
-putInfo :: Text -> ActionT MatchmakerError WebM ()
-putInfo msg = putAssign "flash_alert_info" msg
+putInfo :: Text -> Handler ()
+putInfo msg = void $ putAssign "flash_alert_info" msg
 
-putError :: Text -> ActionT MatchmakerError WebM ()
-putError msg = putAssign "flash_alert_error" msg
+putError :: Text -> Handler ()
+putError msg = void $ putAssign "flash_alert_error" msg
```

Comment:
Let me know if you still don't like the void here, can easily change `putAssign`